code/modules/cargo/supplypod.dm

```diff
@@ -193,7 +193,7 @@
 	else
 		addtimer(CALLBACK(src, .proc/open, src), openingDelay) //After the openingDelay passes, we use the open proc from this supplypod, while referencing this supplypod's contents
 
-/obj/structure/closet/supplypod/open(atom/movable/holder, broken = FALSE, forced = FALSE) //The holder var represents an atom whose contents we will be working with
+/obj/structure/closet/supplypod/open(atom/movable/holder, broken = FALSE, forced = FALSE, force = FALSE) //The holder var represents an atom whose contents we will be working with

```

Comment:
proc redefinitions should have the same arguments as their parents

Reply:
this is an odd proc really. it doesn't  actually use any of the parent vars, and never calls parent. keeping the parent var defs seems like it will make things more jank then necessary

Reply:
Should then be renamed to something like `supplypod_open()`